apps/protocol-frontend/src/components/AttestationsTable.tsx

```diff
@@ -77,6 +77,7 @@ const AttestationsTable = ({
       {
         Header: 'Status',
         accessor: 'status',
+        // @ts-ignore
```

Comment:
If we are adding a bunch of ts-ignores we aren't really solving our typing issues

Reply:
This is actually an issue with react-table v7. It has weak typing. And couldn't be fixed.